_posts/en/newsletters/2022-11-09-newsletter.md

```diff
@@ -0,0 +1,330 @@
+  and an empty `OP_RETURN` output)."
+
+  a5link="https://bitcoincore.reviews/26265#l-100"
+
+  q6="Between disallowing sizes less than 65 bytes and sizes equal to 64 bytes,
+  which approach do you think is better and why?
+  What are the implications of both approaches?"
+
+  a6="After some byte-counting discussion, it was agreed that a valid
+  but non-standard transaction can be as small as 60 bytes:
+  a stripped (non-witness) with a single native segwit input is
+  41 B + 10 B header + 8 B value + 1 B `OP_TRUE` or `OP_RETURN` = 60 B."
+
+  a6link="https://bitcoincore.reviews/26265#l-124"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- Rust Bitcoin 0.28.2 FIXME:harding
+
+- BTCPay FIXME:harding
+
+- [Bitcoin Core 24.0 RC3][] is a release candidate for the
+  next version of the network's most widely used full node
+  implementation.  A [guide to testing][bcc testing] is available.
+
+  **Warning:** this release candidate includes the `mempoolfullrbf`
+  configuration option which several protocol and application developers
+  believe could lead to problems for merchant services as described in
+  this newsletter and previous newsletters [#222][news222 rbf] and
+  [#223][news223 rbf].  It could also cause problems for transaction
+  relay as described in [Newsletter #224][news224 rbf].  Optech
+  encourages any services that might be affected to evaluate the RC and
+  participate in the public discussion.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #26419][] log: mempool: log removal reason in validation interface FIXME:adamjonas
+
+- [Eclair #2404][] adds support for Short Channel IDentifier (SCID)
+  aliases and zero-conf channels (see [Newsletter #156][news156 zc])
+  even for channel state commitments that don't use [anchor
+  outputs][topic anchor outputs].
+
+- [Eclair #2468][] allows the ultimate receiver of a payment ([HTLC][topic
+  HTLC]) to accept a greater amount than they requested and with a
+  longer time before it expires than they requested.  Previously,
+  Eclair-based receivers adhered to [BOLT4][]'s requirement that the
+  amount and expiry delta equal exactly the amount they requested, but
+  that exactitude meant a forwarding node could probe the next hot to
+  see if it was the final receiver by changing either value by the
+  slightest bit.  [BOLTs #1032][] has been opened to add this privacy
```

Comment:
FWIW this has been merged

Reply:
Thanks!  I'm adding BOLTs 1032 to this newsletter so it's easy for these two PR summaries to reference each other.